src/Service/PythonEmbeddingService.php

```
@@ -341,5 +341,63 @@ public function describeImage(string $imagePath): ?string
         }
     }
 
+    public function getActiveEmbeddingModel(): array
```

PHPStan (2.1.15)

Method App\Service\PythonEmbeddingService::getActiveEmbeddingModel() return type has no value type specified in iterable type array.
See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

(missingType.iterableValue)